### PR TITLE
Add basic API docs and FAQs to the README

### PR DESCRIPTION
**Pull request by mbrookes**
Pulled together from various sources (source code, closed issues etc.) Incomplete, but I hope it helps people getting started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,50 +80,194 @@ export default class SimpleMapPage extends Component {
...
 
-###Isomorphic Rendering
-It renders on the server. *(Welcome search engines)* *(you can disable javascript in browser dev tools, and reload any example page to see how it works)*
+####defaultCenter (array or object)
+`[lat, lng]` or `{ lat: lat, lng: ln}`
+Default lat/lng at which to center the map - changing this prop throws a warning
 
-###Component Positions Calculated Independently of Google Maps API
-It renders components on the map before (and even without) the Google Maps API loaded. 
+####defaultCenter (array or object)
```
Comment (istarkov) [inline README.md]: Duplicate of above `####defaultCenter`

File: README.md [hunk 2]
```diff
@@ -54,50 +80,194 @@ export default class SimpleMapPage extends Component {
...
+  return this.props.modalIsOpen
+    ? <GoogleMap />
+    : null;
+}
+```
+
+### Adding a searchbox
+
+```javascript
+mport React from 'react';
```
Comment (istarkov) [inline README.md]: import

**Comment by istarkov** [comment]
@mbrookes I've added you as a collaborator, 
you can merge this and possibly other changes.
Thank you.

**Comment by mbrookes** [comment]
Good spot on defaultCenter & import! I'll fix in a follow-up PR.

**Pull request merged**